eo-runtime/src/test/java/EOorg/EOeolang/EOintTest.java

```diff
@@ -40,7 +40,6 @@
  * @checkstyle TypeNameCheck (4 lines)
  */
 public class EOintTest {
-
```

Comment:
@maxonfjvipon Is this change really necessary?

Reply:
@Graur no, got there by accident. Should I remove it?

Reply:
@maxonfjvipon Yes, It would be great!